lib/cli.js

```diff
@@ -177,16 +177,36 @@ function buildWatcherOptions(options) {
   };
 }
 
-function guardOutputDir(outputDir, removeExisting) {
-  if (fs.existsSync(outputDir)) {
-    if (removeExisting) {
-      rimraf.sync(outputDir);
```

Comment:
Removing rimraf as broccoli already handles doing a diff of the output directory and the new files to be added/removed, so rimraf actually makes it less efficient. https://github.com/broccolijs/broccoli/blob/master/lib/cli.js#L104